addons/skin.confluence/720p/DialogContentSettings.xml

```diff
@@ -60,158 +58,20 @@
 				<textcolor>white</textcolor>
 				<shadowcolor>black</shadowcolor>
 			</control>
-			<control type="label">
-				<description>Used Scraper Header</description>
-				<left>30</left>
-				<top>150</top>
-				<width>320</width>
-				<height>20</height>
-				<font>font12</font>
-				<label>$LOCALIZE[31312]</label>
```

Comment:
It now appears as the button label -
![screenshot004](https://cloud.githubusercontent.com/assets/133808/10840275/6908cec0-7ed5-11e5-8053-9871be5780a7.png)